proxy/core/src/main/scala/io/cloudstate/proxy/protobuf/Utilities.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cloudstate.proxy.protobuf
+
+import com.google.protobuf.descriptor.{FieldOptions => spbFieldOptions, MethodOptions => spbMethodOptions}
+import com.google.protobuf.Descriptors.{FieldDescriptor, MethodDescriptor}
+
+import scala.collection.JavaConverters._
+
+object Types {
+  final val AnyTypeUrlHostName = "type.googleapis.com/"
+}
+
+object Options {
+
+  /**
+   * ScalaPB doesn't do this conversion for us unfortunately.
```

Comment:
- Is it the case that `ScalaPB`-generated code includes an asynchronous client as well as base traits which can be extended to implement server-side? I would think, though, that such being the case would not affect us (in `serverless`, right?)

Reply:
We use this since the binary proto definitions we get are not compiled—they are received at runtime.